build-system/tasks/e2e/index.js

```diff
@@ -153,6 +153,7 @@ e2e.flags = {
   'core_runtime_only': '  Builds only the core runtime.',
   'nobuild': '  Skips building the runtime via `gulp dist --fortesting`',
   'extensions': '  Builds only the listed extensions.',
+  'fast': '  Runs the tests using unminified js',
```

Comment:
I'm not in favor of inventing a new way to run E2E tests. I think they should follow the precedent set by `gulp integration`, gulp serve`, `gulp`, and a bunch of other tasks, all of which work with minified code if `--compiled` is passed in, and on unminified code in the absence of `--compiled`.

The original motivation behind running E2E tests on minified code by default is that most Travis checks should run on minified code if possible. See #22392. This goal can be achieved by passing `--compiled` to `gulp e2e` on Travis.

For local testing, if there's a problem with running on unminified code by default, then `--compiled` should default to true. (In the past, we did run E2E tests on unminified code and it wasn't a problem from what I remember.)

Reply:
All sounds good to me. I'll update with these changes

Reply:
Hmm so how would you test unminified code? `--compiled=false` or the lack of a flag?

Reply:
My vote is to make `e2e` (and later on, `visual-diff`) follow the precedent set by `gulp integration`, `gulp serve`, and `gulp`, where the lack of `--compiled` implies unminified code. SGTY?

Reply:
Oops I missed the part where it would test unminified code by default. OK!